projects/envoyinit/hack/filter_types/filter_types.gen.go

```diff
@@ -327,6 +337,321 @@ import (
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tcp_stats/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/udp_packet_writer/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/generic/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/http/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/tcp/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/udp/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/tcp/generic/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/tcp/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/wasm/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/watchdog/profile_action/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/accesslog/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/auth/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/cluster/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/endpoint/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/event_reporting/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/extension/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/health/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/listener/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/load_stats/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/metrics/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/rate_limit_quota/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/ratelimit/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/redis_auth/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/route/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/runtime/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/secret/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/service/tap/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/type/http/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/type/metadata/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/type/tracing/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/type/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/watchdog/v3"
+
+	// github.com/envoyproxy/go-control-plane/contrib/...
+	_ "github.com/envoyproxy/go-control-plane/contrib/envoy/extensions/compression/qatzip/compressor/v3alpha"
```

Comment:
Somethings gone wrong with the generation here. We are now importing these things twice. Likely due to the module change

Reply:
Some packages are available in the base module (go-control-plane) and the submodules(go-control-plane/contrib, /envoy) etc. 

This should have no actual impact, but we could clean it up by adding a | uniq to the gen script.

On the other hand, it serves as a easy way to see which packages are available in which module.